src/fuel-vm/instruction-set.md

```diff
@@ -462,11 +462,11 @@ Panic if:
 
 `$of` and `$err` are cleared.
 
-### MROO: Math root
+### `MROO`: Math root
 
 |             |                                              |
 |-------------|----------------------------------------------|
-| Description | The (integer) `$rC`th root of `$rB`.         |
+| Description | The (integer) `$rC`ᵗʰ root of `$rB`.         |
```

Comment:
nit: I dislike the fancyness here. Makes it harder to edit.

Reply:
++

Reply:
Would `$rCth` work, or is there some way to reword the sentence? I think it's best to avoid adding "th" in the list of word exceptions, since its a common misspelling of "the".

Reply:
I think we should just add "th" and accept the chance of misspellings